lib/diagnostic.js

```diff
@@ -31,8 +49,20 @@ db.dropDatabase();
 // Daenerys Targaryen
 // Jon Arryn
 
+
+// this one was kind of confusing. Not sure if we ever used $push
 // your code begins here
 
+// looked up $push so I think I would use it like this
+db.houses.insert({ $push: { house_name: "Stark", members: "Ned Stark", "Arya Stark", "Sansa Stark" } });
```

Comment:
You can use `$push` with one argument to add one element or pass in an object with the key that you want to modify and an object value with an `$each` key to add multiple elements.  This is how one might do it with a combination of `$set`, `$push`, and `$push` with `$each`.

```javascript
db.houses.update({
  name: 'House Stark'
}, {
  $set: {
    members: ['Ned Stark']
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $set: {
    members: ['Viserys Targaryen']
  }
});

db.houses.update({
  name: 'House Arryn'
}, {
  $set: {
    members: ['Jon Arryn']
  }
});

db.houses.update({
  name: 'House Stark'
}, {
  $push: {
    members: {
      $each: ['Arysa Stark', 'Sansa Stark']
    }
  }
});

db.houses.update({
  name: 'House Targaryen'
}, {
  $push: {
    members: 'Daenerys Targaryen'
  }
});
```